Sources/TuistKit/Commands/TestCommand.swift

```diff
@@ -65,6 +66,49 @@ struct TestCommand: AsyncParsableCommand {
     )
     var retryCount: Int = 0
 
+    @Option(
+        name: .long,
+        help: "The test plan to run."
+    )
+    var testPlan: String?
+
+    @Option(
+        name: .long,
+        parsing: .upToNextOption,
+        help: "The list of test identifiers you want to test. Takes precedence over --skip-testing",
+        transform: TestIdentifier.init(string:)
+    )
+    var testTargets: [TestIdentifier] = []
+
+    @Option(
+        name: .long,
+        parsing: .upToNextOption,
+        help: "The list of test identifiers you want to skip testing.",
+        transform: TestIdentifier.init(string:)
+    )
+    var skipTestTargets: [TestIdentifier] = []
+
+    @Option(
+        name: .long,
+        parsing: .upToNextOption,
+        help: "The list of configurations you want to test. Takes precedence over --skip-test-configuration"
+    )
+    var testConfigurations: [String] = []
+
+    @Option(
+        name: .long,
+        parsing: .upToNextOption,
+        help: "The list of configurations you want to skip testing."
+    )
+    var skipTestConfigurations: [String] = []
+
+    func validate() throws {
+        let intersection = Set(testTargets).intersection(skipTestTargets)
+        if !intersection.isEmpty {
+            throw ValidationError.invalidTestTargetsOptions(intersection)
+        }
+    }
```

Comment:
There are also other invalid scenario. For example, if testTargets is [`TargetName`] it is invalid/wrong to have skipTestTarget as [`AnotherTargetName/ClassName`], as it doesn't skip anything from the selected set

Reply:
Indeed, though I'm not sure this would be considered an error as technically `AnotherTargetName/ClassName` will be skipped. Maybe we could output some some of warning instead?

Reply:
It's cleary an error in passing the parameter, better to let the user know by making the command fail